gcp/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3.4"
 
 services:
   xk:
-    image: gpii/exekube:0.4.0-google
+    image: gpii/exekube:0.5.0-google_gpii.0
```

Comment:
I'm wondering if we should not use `-` instead of `_`, ie. `0.5.0-google-gpii.0`. All the other images uses `-` as separator for our (`gpii.X`) build suffix.

Reply:
I decided this would be ambiguous in cases where the exekube provided label is already hyphen-separated, e.g. `google-istio`, and chose a different separator.

Reply:
This case is very similar to when I built modified upstream packages for Debian. It uses hyphen (`-`) for separate [the upstream version from the Debian version](https://www.debian.org/doc/debian-policy/ch-controlfields.html#version). Another possibility is to make use of the plus (`+`) because we can understand that is an addition to the upstream package, it's also used in Linux packages when it has a patch on top of the upstream source code.

Reply:
I like +. I'll use that.

Reply:
Heh, never mind -- Docker doesn't allow + in tag names:

https://gitlab.com/gpii-ops/exekube/-/jobs/208845152

```
Error parsing reference: "gpii/exekube:a+b" is not a valid repository/tag: invalid reference format
```